provider/aci.go

```diff
@@ -1632,9 +1635,54 @@ func getProbe(probe *v1.Probe, ports []v1.ContainerPort) (*aci.ContainerProbe, e
 	}, nil
 }
 
+func (p *ACIProvider) getAzureFileCSI(volume v1.Volume, namespace string) (*aci.Volume, error) {
+	azureSource := &aci.AzureFileVolume{
+		ReadOnly: *volume.CSI.ReadOnly,
+	}
+	if volume.CSI.NodePublishSecretRef != nil && volume.CSI.NodePublishSecretRef.Name != "" {
+
+		// Set the storage account name and key
+		secret, err := p.resourceManager.GetSecret(volume.CSI.NodePublishSecretRef.Name, namespace)
+		if err != nil {
+			return nil, fmt.Errorf("The secret %s for AzureFile CSI driver %s is not found, err: %s", volume.CSI.NodePublishSecretRef.Name, volume.Name, err)
```

Comment:
please use errors.Wrap(err, fmt.Sprintf("The secret %s for AzureFile CSI driver %s is not found), volume.CSI.NodePublishSecretRef.Name)

in this way, the caller can still use errors.Is(err,targetErr) to handle different errors throw by the resourceManager